encoders/image/TirgImageEncoder/__init__.py

```diff
@@ -9,6 +9,8 @@
 from jina.executors.encoders.frameworks import BaseTorchEncoder
 from jina.excepts import PretrainedModelFileDoesNotExist
 
+from .img_text_composition_models import TIRG
```

Comment:
Hide this inside function or hide the imports in the file in subfunctions. Otherwise the ugly warning will appear

Reply:
Jina inspects every module when starts and will hit here and import this one and may find that torch is not imported and start complaining

Reply:
What you say is true. When I moved it inside the function, it is not able to import `img_text_composition_models`. Why does VSE has it outside at the top?

Reply:
because I hid the external packages in the `module.py` and so. Ok, I think it is explored in anyway because of the `py_modules`? So maybe we need to hide them in those external files

Reply:
That is doable but what about the current fail message
`could not determine a constructor for the tag '!TirgImageEncoder`

Reply:
Really weird, have u run locally the `jina hub build --prune-images --raise-error --host-info --test-uses --daemon .` from the folder inside this hub?

Reply:
I haven't run it locally as I thought this will work but now it fails. Docker build takes a lot of time on local due to torch download. But I have a workaround for it by adding `pip install torchvision` at the start of dockerfile.

Reply:
does it fail with the same message as here?